dash-renderer/babel.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-    presets: [['@babel/preset-env', {
-        useBuiltIns: 'usage',
-        corejs: 3
-    }], '@babel/preset-react'],
+    presets: [
+        '@babel/preset-env',
```

Comment:
The polyfills are externalized. We only use `preset-env` to know what the transpilation target is.